io.github.picocrypt.Picocrypt.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.picocrypt.Picocrypt
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: picocrypt-gui
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+    - CGO_ENABLED=1
+  build-args:
+    - --share=network
```

Comment:
This is not allowed on Flathub. You have to add the generated manifest instead. This is for generating that manifest "locally"

Reply:
> This is not allowed on Flathub. You have to add the generated manifest instead. This is for generating that manifest "locally"

We have taken this into account, there will be a new commit soon